src/component/NoteModel/NoteModel.js

```diff
@@ -18,13 +19,30 @@ const tailLayout = {
   },
 };
 
-export default function NoteModel({visible = false, onOk = () => {}, setVisible = () => {}}) {
+export default function NoteModel({visible = false, setRefreshNotes = () => {}, setVisible = () => {}}) {
+
+  const [name,setName] = useState('');
+  const [projectName,setProjectName] = useState('');
+  const [note,setNote] = useState('');
 
   const [form] = Form.useForm();
 
   const onFinish = (values) => {
     console.log(values);
-    onOk(values);
+    axios.post("https://nutanix-farewell.herokuapp.com/api/saveNote", {
```

Comment:
Write a file and write API calls there. You should not make calls directly like this from the component